gui/src/app/exampleStanies/exampleStanies.ts

```diff
@@ -26,11 +26,36 @@ model {
     y ~ normal(alpha + beta * x, sigma);
 }
     `.trim()
+
+// generated with beta = 1.25, alpha = -0.1, sigma = 0.1
```

Comment:
The ability to see the means made me realize why this simple linear regression was so slow. The data we were providing was perfectly noiseless, which means sigma was being driven down to zero. But the constraints in stan like `lower=0` are _exclusive_, so you can get 'bouncing' behavior in these cases which slows down HMC.

Adding in a bit of noise means this model now runs very quickly (perhaps _too_ quickly to adequately display the progress/cancellation features -- we can add harder models, if we want that)

Reply:
Nice catch. This does run extremely fast now. Aw shucks, have to use a more powerful example... :smile: